### ISSUE TITLE
problem with proxy

### ISSUE THREAD

**Issue by gioppoluca** [rancher/rancher#17860]
**BUG:**
Trying to execute latest rancher 2.2.0-alpha5

**Steps to reproduce (least amount of steps as possible):**

**Result:**
`2019/02/05 14:44:45 [ERROR] failed to serve UI: Get https://releases.rancher.com/ui/latest2/index.html: dial tcp 104.24.17.51:443: connect: connection timed out
`

**Other details that may be helpful:**

**Environment information**
- Rancher version 2.2.0-alpha5
- single install

I have already set the HTTP_PROXY info as follows

```
[root@rancher ~]# more .docker/config.json
{
 "proxies":
 {
   "default":
   {
     "httpProxy": "http://xxxxx",
     "httpsProxy": "http://xxxxx",
     "noProxy": "localhost,127.0.0.1,xxxxx"
   }
 }
}
[root@rancher ~]# more /etc/systemd/system/docker.service.d/http-proxy.conf
[Service]
Environment="HTTP_PROXY=xxxxx"
Environment="HTTPS_PROXY=xxxxx"
Environment="NO_PROXY=localhost,127.0.0.1,xxxx"

```

**Comment by gioppoluca** [comment]
Rancher web page does not show up but something is going since it present me with the certificate exception of being a self signed one.
Seems that it looks for a non existent page and get stuck there, not being able to serve any interface.

**Comment by gioppoluca** [comment]
this does not happen with 2.1.6 release

**Comment by vincent99** [comment]
Alpha5 is incorrectly pointing at our CDN (which is used intentionally for master build) instead of using a local copy of the UI.  We'll get another tag up soon.

**Comment by cjellick** [comment]
@gioppoluca we just pushed alpha6 which should address that. We are in the process of QAing it ourselves, but i wanted to let you know in case you had time to give it a psin

**Comment by cjellick** [comment]
Note that this bug would also cause alpha5 to always pull the UI from the "lastest2" version off of the CDN, which means it would be constantly updating, which would be bad because the backend code would be fixed at alpha5

**Comment by sangeethah** [comment]
This issue is fixed in `rancher/rancher:v2.2.0-alpha6` release.

When UI loads in js , request URL points to `<rancher server ip>` as expected.

Sample js call made by UI:

`Request URL: https://releases.rancher.com/ui/latest2/assets/vendor.js` — in alpha5
`Request URL: https://<rancher server ip>/assets/ui-2bfa2b2b0c0b42c00b728a1bcae1238a.js` - in alpha6
